app/build.gradle

```diff
@@ -22,16 +22,24 @@ android {
     packagingOptions {
         exclude 'META-INF/rxjava.properties'
     }
+    sourceSets {
+        test.java.srcDirs += 'src/test/kotlin'
+        androidTest.java.srcDirs += 'src/androidTest/kotlin'
+    }
+}
+afterEvaluate {
+    android.sourceSets.all { sourceSet ->
+        if (!sourceSet.name.startsWith("test"))
+        {
```

Comment:
Having the opening brace on a new-line isn't the style we agreed on in the style guide.

Change to `if (!sourceSet.name.startsWith("test")) {`